src/cf-typography/usage.md

```diff
@@ -45,7 +45,7 @@ project by duplicating the variable `@key: value`.
 
 ```
 @webfont-regular: 'AvenirNextLTW01-Regular';
-@webfont-italic: 'AvenirNextLTW01-Italic';
+@webfont-italic: 'AvenirNextLTW01-Regular';
```

Comment:
Would it make sense to update this to:

```
@webfont-italic: @webfont-regular
```

Also, is `@webfont-italic` deprecated? If so, please add `// TODO: Deprecated. Remove in CFv5.` above this. I'm thinking it is if https://github.com/cfpb/capital-framework/issues/700 goes ahead.